crs_application/src/task_managers/process_execution_manager.cpp

```diff
@@ -473,17 +468,36 @@ common::ActionResult ProcessExecutionManager::execTrajectory(const trajectory_ms
     if (status != std::future_status::ready)
     {
       RCLCPP_ERROR(node_->get_logger(), "%s Call Freespace Motion service call timedout", MANAGER_NAME.c_str());
-      return false;
+      return res;
     }
     auto result = result_future.get();
 
     if (!result->success)
     {
       RCLCPP_ERROR(node_->get_logger(), "%s %s", MANAGER_NAME.c_str(), result->message.c_str());
-      return false;
+      return res;
     }
   }
 
+  return true;
+}
+
+common::ActionResult ProcessExecutionManager::execTrajectory(const trajectory_msgs::msg::JointTrajectory& traj)
+{
+  using namespace control_msgs::action;
+  static const double GOAL_ACCEPT_TIMEOUT_PERIOD = 1.0;
```

Comment:
I'd prefer if these were at the top of the cpp file

Reply:
Makes sense, I'll change that

Reply:
Actually that variable wasn't ever used, so I just removed the two instances of it